bin/tachyon

```diff
@@ -177,7 +177,7 @@ function copyDir {
   fi
 
   bin=`cd "$( dirname "$0" )"; pwd`
-  WORKERS=`cat $TACHYON_CONF_DIR/workers`
+  WORKERS=`cat $TACHYON_CONF_DIR/workers | grep -v '^#'`
```

Comment:
All the words of the comment line (including the header) were being passed as workers